src/modules/xcm/index.ts

```diff
@@ -217,6 +217,24 @@ export const xcmChainObj: XcmChainObj = {
     subscan: 'https://bifrost-kusama.subscan.io/',
     isAstarNativeToken: true,
   },
+  [Chain.EQUILIBRIUM]: {
+    name: Chain.EQUILIBRIUM,
+    relayChain: Chain.POLKADOT,
+    img: 'https://contentv2.equilibrium.io/uploads/eq_f2099c1422.svg',
+    parachainId: parachainIds.EQUILIBRIUM,
+    endpoint: 'wss://node.pol.equilibrium.io/',
+    subscan: 'https://equilibrium.subscan.io/',
+    isAstarNativeToken: true,
+  },
+  [Chain.GENSHIRO]: {
+    name: Chain.GENSHIRO,
+    relayChain: Chain.KUSAMA,
+    img: 'https://polkadot.js.org/apps/static/genshiro.ebdc5358..svg',
```

Comment:
This URL doesn't work anymore
![image](https://user-images.githubusercontent.com/92044428/223325412-a72228b0-bacb-42d7-92b8-f1c6523adc86.png)